test/jdk/jdk/internal/vm/TestTranslatedException.java

```diff
@@ -152,13 +162,15 @@ private void encodeDecode(Throwable throwable) throws Exception {
         }
     }
 
-    private static void assertThrowableEquals(Throwable original, Throwable decoded) {
+    private static void assertThrowableEquals(Throwable originalIn, Throwable decodedIn) {
+        Throwable original = originalIn;
+        Throwable decoded = decodedIn;
```

Comment:
What is the purpose of this renaming?

Reply:
So that the printing down the bottom of this message shows the complete throwable, not just the cause on which the comparison failed.

Reply:
Thanks! I missed the reassign in the folded unchanged code.